proto/compiler.bzl

```diff
@@ -93,6 +98,17 @@ def go_proto_compile(go, compiler, protos, imports, importpath):
                 ext = compiler.internal.suffix,
             )
             go_srcs.append(out)
+
+            if extra_outputs.get(src.basename):
```

Comment:
You can assign this to a variable so that you don't need to `get` it again in the next line.